component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java

```diff
@@ -249,9 +249,10 @@ public Response createSpace(@Context UriInfo uriInfo,
     space.setMembers(members);
 
     //
-    spaceService.createSpace(space, authenticatedUser, model.getInvitedMembers());
+    Space createdSpace = spaceService.createSpace(space, authenticatedUser, model.getInvitedMembers());
+    spaceService.addRedactor(createdSpace, authenticatedUser);
```

Comment:
This is a business logical, so it should be handled in Service layer not in REST. In addition, this should be added in the same place where the space creator is promoted as manager by default.

Reply:
ok, i will make the changes

Reply:
yes normally @hassenelkamel-exo we should not touch the space creation, we should only change the condition of post button display in order to be always accessible to space managers